README.md

```diff
@@ -14,362 +14,49 @@ Install the plugins with your preferred plugin manager. For example, with [`vim-
 Plug 'MunifTanjim/nui.nvim'
 ```
 
-## Usage
+## Components
 
 ### Popup
 
-Popup is an abstraction layer on top of window.
-
-Creates a new popup object (but does not mount it immediately).
+![Popup GIF](./media/pop.gif)
 
 ```lua
 local Popup = require("nui.popup")
+local event = require("nui.utils.autocmd").event
 
 local popup = Popup({
-  border = {
-    style = "rounded",
-    highlight = "FloatBorder",
-  },
-  position = "50%",
-  size = {
-    width = "80%",
-    height = "60%",
-  },
+      enter = true,
```

Comment:
Let's use the same indentation as the codebase (2 spaces).

Reply:
Makes sense, will fix.